### ISSUE TITLE
Can the codelens sample be updated to include the newly available Custom UI example?

### ISSUE THREAD

**Issue by StingyJack** [microsoft/VSSDK-Extensibility-Samples#180]
The 16.1 release notes mention "There is now CodeLens support for custom UI." , but there isnt any documentation I can find about it. 

https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes

**Comment by IntelOrca** [comment]
I don't think it is possible without using internal APIs. I have raised an issue on dev community:

https://developercommunity.visualstudio.com/content/problem/619055/code-lens-custom-ui-can-not-be-created-for-code-le.html

**Comment by IntelOrca** [comment]
#181 got merged. It appears you can inject small bits of custom UI before and after the table in Code Lens.
